source/renderer/app/containers/staking/StakingEpochsPage.js

```diff
@@ -12,6 +12,16 @@ export default class StakingEpochsPage extends Component<Props> {
   static defaultProps = { actions: null, stores: null };
 
   render() {
-    return <StakingEpochs name="StakingEpochs" />;
+    return (
+      <StakingEpochs
+        currentEpochName=""
+        currentEpochData={[]}
+        currentEpochEndDateTime="2019-12-31"
```

Comment:
@yakovkaravelov end date time should contain time too. It will be a the same as here: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/stores/StakingStore.js#L10